### PR TITLE
Change course

### PR DESCRIPTION
**Pull request by simonwhatley**

### CODE REVIEW

File: app/data/helpers/courses.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
+const Utils = require('./utils')
```
Comment (adamsilver) [inline app/data/helpers/courses.js:1]: if Utils isn't a constructor then I wouldn't capitalise. Nitpick alert.

File: app/data/helpers/courses.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+const courses = require('../courses')
+
+exports.getCourses = (selectedItem) => {
+  const items = []
+
+  courses.forEach((course, i) => {
+    const item = {}
+
+    item.text = course.name
+    item.text += ' (' + course.code + ')'
```
Comment (adamsilver) [inline app/data/helpers/courses.js:13]: Thinking that the combining should happen in the view as it's view-y stuff.

File: app/data/helpers/courses.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    item.id = studyMode
+    item.checked = (selectedItem && selectedItem.includes(studyMode)) ? 'checked' : ''
+
+    items.push(item)
+  })
+
+  return items
+}
+
+exports.getCourseLocations = (courseId, selectedItem) => {
```
Comment (adamsilver) [inline app/data/helpers/courses.js:58]: To me these methods are about getting data, not manipluating it to be ready for the view.

File: app/routes/edit-course.js [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+    if (req.query.referrer === 'check') {
+      back += '/course/edit/check'
+    }
+
+    res.render('applications/course/course', {
+      application: req.session.data.applications.find(app => app.id === req.params.applicationId),
+      courses: CourseHelper.getCourses(selectedCourse),
+      actions: {
+        back: back,
+        cancel: `/applications/${req.params.applicationId}/course/edit/cancel`,
```
Comment (adamsilver) [inline app/routes/edit-course.js:22]: Why is cancel a separate url?

Comment (simonwhatley) [reply app/routes/edit-course.js:22]: For convenience to clear out the session data.

File: app/data/helpers/courses.js (hunk 1 above)
Comment (simonwhatley) [reply app/data/helpers/courses.js]: Changed to lower case

**Pull request merged**